docs/source/tutorials/07_using_model_hub_nodes.rst

```diff
@@ -0,0 +1,334 @@
+.. include:: /include/substitution.rst
+
+*********************
+Using Model Hub Nodes
+*********************
+
+PeekingDuck has support for external model hub models. You can leverage the model
+hub nodes to use these external models with the PeekingDuck pipeline. The model
+hub nodes differ in design from traditional model nodes as the model hub models
+perform a variety of computer vision tasks. This tutorial demonstrates how
```

Comment:
Suggest rephrasing 3rd line as `The model hub nodes differ in design from traditional model nodes as the
external models perform a variety of computer vision tasks.`

Reply:
Changed